website/routes.py

```diff
@@ -882,6 +882,24 @@ def make_url_map(app):
             project_views.register.node_registration_embargo_disapprove,
             OsfWebRenderer('error.mako', trust=False)
         ),
+        Rule(
```

Comment:
These aren't API URLs, so they aren't normally what I would review. I'm happy to pass judgement, however.